protoc-gen-orion/internal/ports/inputs/protoc.go

```diff
@@ -0,0 +1,122 @@
+package inputs
```

Comment:
The inputs package decouples the dependency of protobuf library from internal logic.